src/Api/AppCatalog/AppCatalogApi.php

```diff
@@ -0,0 +1,76 @@
+<?php
+declare(strict_types=1);
+
+namespace Akeneo\Pim\ApiClient\Api\AppCatalog;
+
+use Akeneo\Pim\ApiClient\Client\ResourceClientInterface;
+use Akeneo\Pim\ApiClient\Exception\InvalidArgumentException;
+use Akeneo\Pim\ApiClient\Pagination\PageFactoryInterface;
+use Akeneo\Pim\ApiClient\Pagination\PageInterface;
+use Akeneo\Pim\ApiClient\Pagination\ResourceCursorFactoryInterface;
+use Akeneo\Pim\ApiClient\Pagination\ResourceCursorInterface;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license https://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class AppCatalogApi implements AppCatalogApiInterface
+{
+    const APP_CATALOGS_URI = 'api/rest/v1/catalogs';
+    const APP_CATALOG_URI = 'api/rest/v1/catalogs/%s';
+    const ID_MUST_BE_DEFINED_EXCEPTION_MESSAGE = 'The parameter "id" should not be defined in the data parameter';
```

Comment:
*must* not be defined, since it's blocking

Reply:
I removed ID_MUST_BE_DEFINED_EXCEPTION_MESSAGE since it is not so useful. Other constants are in public, so the visibility is 'public', but do you prefer an explicit one and I add the` public` keyword?